sdk/keyvault/keyvault-secrets/package.json

```diff
@@ -20,7 +20,7 @@
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
   "main": "./dist/index.js",
-  "module": "dist-esm/keyvault-secrets/src/index.js",
+  "module": "dist-esm/src/index.js",
```

Comment:
@bzhang0 you shouldn't need to change this line.

KeyVault and Storage are special in that they use shared source from a common folder which leads to this directory structure.

Reply:
Ah, I see - it seems like doing ```rushx lint --fix``` probably simplified that line down. I will try again without making that change!